superset-frontend/src/views/CRUD/flash/constants.ts

```
@@ -64,3 +64,109 @@ export const ERROR_MESSSAGES = {
   REMOTE_SERVER:
     'Unable to connect with the remote server. Please contact your Administrator or communicate on the required Support Channel.',
 };
+
+export const FLASH_TYPE_JSON = {
+  JSONSCHEMA: {
+    type: 'object',
+    properties: {
+      flashType: {
+        title: 'Flash Type',
+        type: 'string',
+        enum: ['', 'One Time', 'Short Term', 'Long Term'],
```

Do we really need this JSON if you can get the Flash Types from backend? Just like Status

---

We are getting it from backend but this is for two reasons:
1- fallback case (if it failed to catch data)
2- For testing, there should be a default case because react DOM doesn't persist data when independent component is rendered.

---

1. If the fetch fails then let's not fix this on UI. Let it fail. Nothing should be assumed on UI.
2. For testing you can assume this JSON in test class and remove this JSON from here.

---

Created ticket in backlog for this task